homeassistant/components/sensor/gearbest.py

```diff
@@ -19,6 +19,9 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ITEMS = 'items'
```

Comment:
Good idea to factor these out. However, at least `CONF_NAME`, `CONF_URL`, `CONF_ID` and `CONF_CURRENCY` should be imported from `homeassistant/const.py`.

Reply:
Thanks for pointing this out! What do you think about using CONF_ENTITIES instead of items?

Reply:
In the context of home assistant, `entity` is a very fixed term for an object representing a physical (or virtual) device, something that has a state, belongs to a component, offers services, can be represented in the front end, etc. So, no, don't use that term for Gearbest items. ;-)